WordPress/Classes/Services/PostService.m

```diff
@@ -206,7 +221,7 @@ - (void)syncPostsOfType:(NSString *)postType
                    success:^(NSArray *posts) {
                        BOOL hasMore = ([posts count] < PostServiceNumberToFetch) ? NO : YES;
                        [self.managedObjectContext performBlock:^{
-                           [self mergePosts:posts ofType:postType withStatuses:postStatus forBlog:blog purgeExisting:YES completionHandler:^{
+                           [self mergePosts:posts ofType:postType withStatuses:postStatus byAuthor:authorID forBlog:blog purgeExisting:YES completionHandler:^{
```

Comment:
Not purging would create problems when loading more.  It *is* bandwidth expensive, but the alternative is we end up with gaps in content when the oldest "only me" post is older than newer "everyone" posts that have not yet been synced.